recipe-server/normandy/settings.py

```diff
@@ -110,6 +111,25 @@ class Core(Configuration):
         }
     }
 
+    # Content Security Policy
+    def CSP_DEFAULT_SRC(self):
+        srcs = ["'self'"]
+        if self.CDN_URL:
+            srcs.append(self.CDN_URL)
+        return srcs
+    CSP_OBJECT_SRC = "'none'"  # not using <object>, <embed>, and <applet> elements
```

Comment:
nit: Shouldn't there be a blank line between the method and `CSP_OBJECT_SRC`? I mean, the linter didn't care so it is fine, but it looks weird.